Examples/ObjectiveC/MultipleImagesExample.m

```diff
@@ -0,0 +1,51 @@
+
+#import "MultipleImagesExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleMultipleImages = @"MultipleImagesExample";
+
+@interface MultipleImagesExample () <MGLMapViewDelegate>
+
+@end
+
+@implementation MultipleImagesExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    // Create and add a map view.
+    MGLMapView *mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds styleURL:[MGLStyle outdoorsStyleURL]];
+    
+    // Center the map on Yosemite National Park, United States.
+    [mapView setCenterCoordinate:CLLocationCoordinate2DMake(37.761, -119.624) zoomLevel:10 animated:NO];
+    mapView.autoresizingMask = UIViewAutoresizingFlexibleHeight | UIViewAutoresizingFlexibleWidth;
+    mapView.delegate = self;
+    [self.view addSubview:mapView];
+}
+
+- (void)mapView:(MGLMapView *)mapView didFinishLoadingStyle:(MGLStyle *)style {
+    // Add icons from the U.S. National Parks Service to the map's style.
+    [style setImage:[UIImage imageNamed:@"nps-restrooms"] forName:@"restrooms"];
+    [style setImage:[UIImage imageNamed:@"nps-trailhead"] forName:@"trailhead"];
+    [style setImage:[UIImage imageNamed:@"nps-picnic-area"] forName:@"picnic-area"];
+    
+    // Access a vector tileset that contains places of interest at Yosemite National Park. This tileset was created by uploading NPS shapefiles to Mapbox Studio.
+    NSURL *url = [[NSURL alloc] initWithString:@"mapbox://examples.ciuz0vpc"];
+    MGLVectorTileSource *source = [[MGLVectorTileSource alloc] initWithIdentifier:@"yosemite-pois" configurationURL:url];
+    [style addSource:source];
+    
+    MGLSymbolStyleLayer *layer = [[MGLSymbolStyleLayer alloc] initWithIdentifier:@"yosemite-pois" source:source];
+    
+    // Access the layer that contains the POI data.
```

Comment:
It would be useful to clarify what the source layer identifier is (it is the unique identifier of the particular layer within a vector source), and [how it can be obtained](https://www.mapbox.com/api-documentation/#retrieve-tilejson-metadata).